common/lib/xmodule/xmodule/tests/test_split_test_module.py

```diff
@@ -524,3 +526,87 @@ def verify_summary_message(general_validation, expected_message, expected_messag
             u"This content experiment has issues that affect content visibility.",
             StudioValidationMessage.ERROR
         )
+
+
+class SplitTestBlockExportImportTest(MixedSplitTestCase):
```

Comment:
Great test @symbolist . Were you able to run it successfully before the conversion?

Reply:
Yes, it works with the XModule version as well.